src/core/transaction.zig

```diff
@@ -576,61 +583,54 @@ test "create transfer transaction" {
 }
 
 test "tmp" {
```

Comment:
I don't think this test is needed anymore.

Reply:
adf81248948f26675220f5dd8c4cad753d53f37d